www/js/directives/sf_resize.js

```diff
@@ -0,0 +1,21 @@
+app.directive('sfResize', function ($window) {
+  return function (scope, element) {
+    scope.getWinHeight = function() {
+      return $window.innerHeight;
+    }
+
+    var setNavHeight = function(newHeight) {
```

Comment:
I did not write this from scratch. Used someone else's solution. Cleaned this up a bit.